jni/src/org_opensearch_knn_jni_NmslibService.cpp

```diff
@@ -12,79 +12,102 @@
 #include "org_opensearch_knn_jni_NmslibService.h"
 
```

Comment:
Please ignore the formatting changes and directly go to `Java_org_opensearch_knn_jni_NmslibService_loadIndexWithStream`.